### PR TITLE
OpenEXROutput: skip dwa compression for single channel small tile images

### PR DESCRIPTION
**Pull request by rmv**
dwa? compression fails when saving single channel files with small tile sizes (< 16)

Larry determined in #1844 that the underlying issue is in OpenEXR.

While that gets fixed we should skip dwa compression for the failure cases. This patch replaces dwa compression with the default zip for the failing cases.

### CODE REVIEW

File: src/openexr.imageio/exroutput.cpp [hunk 1]
```diff
@@ -679,6 +679,11 @@ OpenEXROutput::spec_to_header(ImageSpec& spec, int subimage,
     // on deep files (but allow "none" as well)
     if (spec.deep && comp != "none")
         comp = "zips";
+    // Currently dwaa/b compression does not work on single channel with tile < 16
+    if (spec.nchannels == 1 && spec.tile_width < 16 && spec.tile_height < 16 &&
```
Comment (lgritz) [inline src/openexr.imageio/exroutput.cpp]: Is it if *both* width and height are < 16? Or if either one is?
I don't know/remember, just asking.

**Comment by lgritz** [comment]
LGTM. I'll await your answer to the one question I had, and for the CI tests to return successfully. Then I'll merge. Thanks for the fix.

**Comment by lgritz** [comment]
Fixes #1844

**Comment by rmv** [comment]
For instance 31x15 and 16x8 fail, but 31x16 does not

The patch does not fix all the underlying failure conditions with dwa compression, it fixes the ones that affect maketx (8x8, 4x4, etc), so:

Fixes #1844

**Comment by lgritz** [comment]
I'm not sure we should knowingly still have a bunch of failure cases.

Should we just consider dwa + 1-channel to be unsafe for anything other than power-of-2 square tile sizes >= 16? And make everything else automatically revert to zip, until we know there's a robust fix on the OpenEXR side? (Keeping in mind that this will still allow dwaa for all the maketx and other uses that we truly care about.)

**Comment by rmv** [comment]
Sounds good, there were still quite a lot of failing combinations for non power of 2 tile sizes under 64x64, see new patch.

**Comment by lgritz** [comment]
LGTM, I will merge after the CI tests come back.

**Comment by rmv** [comment]
Thanks!

**Pull request merged**